packages/contracts/as/NFT/assembly/index.ts

```diff
@@ -322,6 +327,93 @@ export function nft_revoke_all(token_id: string): void {
     }
 }
 
+/* Storage management */
```

Comment:
Could we extract this section as much as possible into its separate file/module for organization and also to make it easy to merge even if it's not in its final state.